hydractl/build-ghaf-branch.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: 2022-2023 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+
+# Stop on error
+set -e
+
+HCTL="python3 hydractl.py https://hydra2.vedenemo.dev/"
```

Comment:
Especially as this PR is targeted to mainline (and not hydra2 maintenance branch), there should be no hardcoded hydra2 instance URL here.

You could, e.g., source "containers/confs/hydra.default" and use "${HYDRA_URL}" it sets (if any).
Alternatively, as this now lives under hydractl/ and not containers/, would be to require the URL as a command line parameter.

Reply:
Then make it more clearly as "just an example". E.g. I'd accept either of these:

- Place it in an "examples" subdirectory: "hydractl/examples/"
- Add explanatory comments to the script, on top, first thing after the copyright lines.